Userland/Applications/Browser/Tab.cpp

```diff
@@ -135,6 +135,22 @@ Tab::Tab(BrowserWindow& window, Type type)
         view().set_focus(true);
     };
 
+    m_location_box->on_ctrl_return_pressed = [this] {
+        auto trimmed_text = m_location_box->text().trim("."sv, TrimMode::Both);
```

Comment:
What is the reason for trimming here?

Reply:
Trimming periods off the start and end lets us construct the URL with `.com` easier and use `.contains("."sv)` to do a dumb check for a tld.

> e.g. "example." or ".example" -> "example.com"

> e.g. "example.faketld" or "example.ch" or "sub.example.ch" -> follow URL as given

Reply:
Good thought.